universal-application-tool-0.0.1/app/views/admin/questions/QuestionConfig.java

```diff
@@ -230,21 +239,34 @@ private QuestionConfig addMultiOptionQuestionFields(
         multiOptionQuestionForm.getOptionIds().size()
             == multiOptionQuestionForm.getOptions().size(),
         "Options and Option Indexes need to be the same size.");
-    ImmutableList.Builder<ContainerTag> existingOptionsBuilder = ImmutableList.builder();
+    ImmutableList.Builder<ContainerTag> optionsBuilder = ImmutableList.builder();
+    int optionIndex = 0;
     for (int i = 0; i < multiOptionQuestionForm.getOptions().size(); i++) {
-      existingOptionsBuilder.add(
+      optionsBuilder.add(
           multiOptionQuestionField(
               Optional.of(
                   LocalizedQuestionOption.create(
                       multiOptionQuestionForm.getOptionIds().get(i),
-                      i,
+                      optionIndex,
                       multiOptionQuestionForm.getOptions().get(i),
                       LocalizedStrings.DEFAULT_LOCALE)),
-              messages));
+              messages,
+              false));
```

Comment:
nit: include param name comment
```suggestion
              /* isForNewOption= */ false));
```